### PR TITLE
Chakra setup

### PR DESCRIPTION
**Pull request by avinashi10**
## Description

This code sets up Chakra UI as the CSS library that will be used to style this app. It extends Chakra's default theme with custom colors and fonts. It also applies some Chakra features to ensure that the setup is working correctly.

## Acceptance Criteria

Chakra UI components should be able to be successfully imported in each React component.

## Type of Changes

|     | Type                       |
| --- | -------------------------- |
|     | :bug: Bug fix              |
|     | :sparkles: New feature     |
|     | :hammer: Refactoring       |
|     | :100: Add tests            |
|  ✓  | :link: Update dependencies |
|     | :scroll: Docs              |
| ✓   | :sparkling_heart: Styling     |

### Before
Screenshot 2023-09-28 at 4 35 45 PM

### After
Screenshot 2023-09-28 at 4 34 39 PM

## Testing Steps / QA Criteria

1. Switch to the `chakra-setup` branch on your local computer
2. Run `npm i` to install the newest dependencies
3. Run `npm start` to start the application
4. In your web browser ensure that the app features are still working. Verify that some custom colors and fonts are visible as per the agreed upon style guide.
5. Clear tokens from local storage and check that features of home page and working and verify that some styling is visible.
6. If you create new lists, be sure to delete them from Firestore to keep the database easy to navigate.

### CODE REVIEW

**Review by hajschroeder — APPROVED** [review]
Looks good! New dependencies installed no problem, and the styling is off to a great start

**Review by codecaviette — APPROVED** [review]
LGTM!

**Review by MagdaSlifierz — APPROVED** [review]
Looks great.  👍

File: src/components/ListItem.jsx [hunk 1]
```diff
@@ -88,7 +90,15 @@ export function ListItem({
...
 			<label htmlFor={name}> {name} </label>
 			<div className="urgency-indicator">{urgency}</div>
-			<button onClick={handleDelete}> Delete </button>
+			<Button
+				leftIcon={<DeleteIcon />}
+				bg="brand.navy"
+				color="brand.off_white"
+				onClick={handleDelete}
+			>
+				{' '}
```
Comment (jeremiahfallin) [inline src/components/ListItem.jsx]: The spaces are unnecessary here.

File: src/views/Home.jsx [hunk 2]
```diff
@@ -1,9 +1,10 @@
 // LIBRARY IMPORTS
 import { useNavigate } from 'react-router-dom';
 import React, { useState } from 'react';
+import { Flex, Spacer, Button } from '@chakra-ui/react';
 
 // LOCAL IMPORTS
-import './Home.css';
```
Comment (jeremiahfallin) [inline src/views/Home.jsx:6]: Removed default css here but not in other places. Good that you're all thinking about how the css from the Chakra library interacts with the css that was in the project previously.

**Review by jeremiahfallin — APPROVED** [review]
lgtm!

File: src/components/ListItem.jsx (hunk 1 above)
Comment (avinashi10) [reply src/components/ListItem.jsx]: removed these!

File: src/views/Home.jsx (hunk 2 above)
Comment (avinashi10) [reply src/views/Home.jsx:6]: Good call, there's still previous styling in `ListItem.jsx`/`ListItem.css`. We just added issues for styling and the person who takes on styling the list items can replace these styles and refer to them if needed. I deleted the `Home.css` and `Layout.css` files to keep the repo tidy

**Review by lyleschemmerling — APPROVED** [review]
lgtm

**Review by thetrend — APPROVED** [review]
Looks great! 🥳

**Pull request merged**